src/s_print.c

```diff
@@ -11,6 +11,21 @@
 #include "s_stuff.h"
 #include "m_private_utils.h"
 
+#ifdef _WIN32
+    #ifdef _MSC_VER
+        #define PD_FWPRINTF_NARROW_FORMATTER L"%S"
+    #else
+        #if __USE_MINGW_ANSI_STDIO
+            #define PD_FWPRINTF_NARROW_FORMATTER L"%s"
+        #else
+            #define PD_FWPRINTF_NARROW_FORMATTER L"%S"
+        #endif
+    #endif
+#endif
+#ifndef PD_FWPRINTF_NARROW_FORMATTER
+    #define PD_FWPRINTF_NARROW_FORMATTER L"%S"
+#endif
```

Comment:
> If you want to allow overriding the format specifier,

the last ifndef was more in case someone would use fwprintf on some potentially other libc than msvcrt / urcrt (plibc comes to mind, I guess cygwin can use glibc too?) at some point and reuse this macro, but I assume this is not going to happen or be a priority